src/types.ts

```diff
@@ -211,3 +211,20 @@ export type AppClient = {
    */
   requirePkce: boolean;
 };
+
+export type UserInfo = {
+  name: string;
+  middleName?: string;
+  givenName?: string;
+  phoneNumber?: string;
+  phoneNumberVerified?: boolean;
+  email?: string;
```

Comment:
This type should only have the information that was actually added in the database migration. Things like `phoneNumber` and `email` exists in other part of the datamodel (it's in PrincipalIdentity).

Reply:
good callout! removed!